recipes/ilastik-launch/meta.yaml

```diff
@@ -3,9 +3,8 @@ package:
   version: 0.3
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   run:
-    - ilastik-core
-    - volumina
+    - ilastik
```

Comment:
ideally the minumum requirement would be `>=1.4.0b29` if you choose to also change the app name in the run_ilastik.sh...